casper/src/test/scala/coop/rchain/casper/batch2/RholangBuildTest.scala

```diff
@@ -1,80 +0,0 @@
-package coop.rchain.casper.batch2
-
-import cats.syntax.all._
-import coop.rchain.casper.genesis.contracts.Vault
-import coop.rchain.casper.helper.TestNode
-import coop.rchain.casper.helper.TestNode._
-import coop.rchain.casper.rholang.Tools
-import coop.rchain.casper.util.ConstructDeploy
-import coop.rchain.casper.util.GenesisBuilder._
-import coop.rchain.casper.util.RSpaceUtil._
-import coop.rchain.crypto.signatures.Secp256k1
-import coop.rchain.rholang.interpreter.util.RevAddress
-import coop.rchain.shared.{Base16, RChainScheduler}
-import coop.rchain.shared.scalatestcontrib._
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers
-
-class RholangBuildTest extends AnyFlatSpec with Matchers {
```

Comment:
It's difficult to decipher all the nonsense I seen in _unit_ tests. This is just another example where description of the test doesn't correspond with what's testing.
So in this test, it's just repeats what is done in most of the tests so it doesn't contribute anything.

There are a few major categories of tests that doesn't make sense:
- it's not a unit test because it's testing overall behavior of the node (most of the tests)
- it's testing functionality not important for the tests or completely missing the testing subject
  - e.g. [GenesisTest](https://github.com/rchain/rchain/blob/15d631623e1f6d10ec53c306338541a5402c61a4/casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala) which is testing parsing of bonds file which is not related to genesis tests
  - BlockDagStorageTest (removed is this PR) which tests data stored on disk which is not part of storage functionality
- it's testing very specific data like data in the block but test build much more data not relevant for testing
  - e.g. [ValidateTest](https://github.com/rchain/rchain/blob/15d631623e1f6d10ec53c306338541a5402c61a4/casper/src/test/scala/coop/rchain/casper/batch2/ValidateTest.scala) is building the DAG to just check the functions operating on the block data